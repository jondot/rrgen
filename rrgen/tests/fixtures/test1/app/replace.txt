// first hello should be replaced with world

pub struct Hello {
  pub name: String,
}

pub struct Hello {
  pub name: String,
}
