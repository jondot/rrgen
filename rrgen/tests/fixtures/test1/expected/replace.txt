// first hello should be replaced with world

pub struct World {
  pub name: String,
}

pub struct Hello {
  pub name: String,
}
